source/manual/grafana.html.md

```diff
@@ -5,11 +5,20 @@ section: Monitoring
 type: learn
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-02-26
+last_reviewed_on: 2020-06-30
 review_in: 6 months
 ---
 
-Grafana lets us create nice dashboards using Graphite data.
+[Grafana](https://grafana.com/) is an open-source visualisation tool.
+It does not store data, but consumes data sources to create real-time
+graphs displayed on custom dashboards. Data sources include Prometheus,
+[Graphite], [Logit] and CloudWatch. The query language of the data
```

Comment:
I think we need extra square brackets here, such as [Graphite][].

Reply:
[Shouldn't need to](https://github.github.com/gfm/#example-565) (we use this style in a few other places). (I only discovered that myself a couple of months ago - it's not intuitive at first!)